--- git/CMakeLists.txt	2017-08-17 13:44:32.851289264 +0000
+++ git/CMakeLists.txt	2017-08-17 13:41:31.938923452 +0000
@@ -13,7 +13,7 @@
 endif()
 
 find_package(console_bridge REQUIRED)
-find_package(poco_vendor REQUIRED)  # Provides FindPoco.cmake and Poco on platforms without it.
+#find_package(poco_vendor REQUIRED)  # Provides FindPoco.cmake and Poco on platforms without it.
 find_package(Poco REQUIRED COMPONENTS Foundation)
 link_directories(${Poco_LIBRARY_DIR})
 
